core/last_callers.js

```diff
@@ -74,11 +74,25 @@ exports.getModule = class LastCallersModule extends MenuModule {
                                     return false === User.isRootUserId(parseInt(lh.log_value)); //  log_value=userId
                                 });
 
+                                var prevUserId = 0;
```

Comment:
Pretty much never need to use `var` anymore - use `let`.

Reply:
Closing this since another system was implemented. If it's missing something that was here, lemme know!